src/components/Header.tsx

```diff
@@ -62,10 +62,12 @@ export const Header: FC = memo(props => {
       <Menu>
         <MenuItem onClick={handleBookmarksVisibilityClick}>
           {isShowingHiddenBookmarks ? <StyledHideIcon /> : <StyledShowIcon />}
+          <ToolTip >{isShowingHiddenBookmarks ? 'Hide hidden bookmarks' : 'Show hidden bookmarks'}</ToolTip>
```

Comment:
Aaaand opening tag here.
Please also check this code with `prettier`, this line is longer than 80 symbols, I believe this project uses default (?) prettier settings, where max line length is 80 symbols. 

```suggestion
          <ToolTip>
            {isShowingHiddenBookmarks ? 'Hide hidden bookmarks' : 'Show hidden bookmarks'}
          </ToolTip>
```

Reply:
Yep, that's correct! I haven't forced it though so @ONeill45 if you have problems using Prettier let me know and I'll do it after the merge.